app/models/keyword.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Keyword < ApplicationRecord
+  enum status: { processing: 'processing', succeeded: 'succeeded', failed: 'failed' }
+  has_many :keyword_search_entries, dependent: :destroy
+  validates :content, :status, presence: true
```

Comment:
Again here, line break issues:

```suggestion
	enum status: { processing: 'processing', succeeded: 'succeeded', failed: 'failed' }
	
  has_many :keyword_search_entries, dependent: :destroy
  
  validates :content, :status, presence: true
```

Make sure to follow this structure for all models: https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#models

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/b2aef75f8792a54884596c0bd2e63c805121f4ff